src/main/java/org/fundacionjala/trello/pages/home/sections/TeamSection.java

```diff
@@ -0,0 +1,21 @@
+package org.fundacionjala.trello.pages.home.sections;
+
+import org.openqa.selenium.WebDriver;
+
+public final class TeamSection extends BodySection {
+
+    public TeamSection(final WebDriver driver, final String name) {
+        super(driver, name);
+    }
+
+    @Override
+    protected void initialize(final String section) {
+        baseSelector = String.format("//h3[contains(text(),'%s')]", section);
+        sectionNameSelector = baseSelector;
+
```

Comment:
Is this necessary?

Reply:
Whenever I leave a blank space it is to divide the context, in some cases it could be difficult to read if everything is joined.